src/tests/ftest/pool/multi_server_create_delete_test.py

```diff
@@ -123,12 +118,11 @@ def test_create(self):
 
         try:
             cmd = (
-                "{} create "
-                "--mode={} "
-                "--uid={} "
-                "--gid={} "
+                "{} pool create "
```

Comment:
Something to consider:  Makito is enhancing dmg_utils.py in https://github.com/daos-stack/daos/pull/1307.  With his changes the `dmg pool create` and `dmg pool destroy` actions would probably be better implemented through a `DmgCommand` object.